packages/@okta/i18n/src/properties/login.properties

```diff
@@ -685,8 +685,11 @@ deviceTrust.universalLink.fallback.getOktaMobile.subtitle = Go to the {0}AppStor
 loopback.polling.cancel.link = Cancel and take me to sign in
 customUri.subtitle = Launching Okta Verify...
 customUri.content = <div class="skinny-content"> If nothing prompts from the browser, <a href="#" id="launch-ov" class="link">click here</a> to launch Okta Verify, or make sure Okta Verify is installed.</div>
-customUri.required.content.p1 = Please click&nbsp;<span class="strong">Open oktaverify.app</span>&nbsp;if you see the system dialog.
-customUri.required.content.p2 = If nothing prompts from the browser,&nbsp;<a href="#" id="launch-ov" class="link">click here</a>&nbsp;to launch Okta Verify, or&nbsp;<a href="{0}" target="_blank" id="download-ov" class="link">download & run Okta Verify</a>.
+customUri.title = Click "Open Okta Verify" on the browser prompt
```

Comment:
Do we need to add `prompt` here `browser prompt` ? I feel `Click "Open Okta Verify" on the browser` should be just fine.

Reply:
@panchamyadav-okta can you please answer that?

Reply:
Hi @pradeepdewda-okta! thanks for the feedback! Discussed this with the design team, and we agree that "Click 'Open Okta Verify' on the browser prompt" makes the intended action clearer to the user than "on the browser", which could be a little confusing since the user wouldn't know where to expect the "Open OV" message on the browser.